src/AppBundle/Controller/Admin2/MatchController.php

```diff
@@ -165,27 +185,54 @@ public function approveAction(Request $request)
 
         throw $this->createNotFoundException();
     }
-
+	/**/public function listCommonCategoriesbyLocale(User $user, $locale)
+	{
+		$categories = [];
+		$matchUser = $this->categoryManager->getFindByIdsAndLocale(array_keys($user->getCategoryNames()), $locale);
+		$userCategories        = $matchUser->getCategoryNames();	
+		foreach($matchUser as $matchUserCategory) {
+            if (isset($userCategories[$matchUserCategory->getId()])) {
+                $categories[] = $matchUserCategory->getName();				
+            } 
+        }
+		if (count($categories) > 1) {
+            $lastCategory      = array_pop($categories);
+            $categories     = implode(', ', $categories) .' '.  $this->translator->trans('global.and') .' '. $lastCategory;
+		} else {
+            $categories     = implode(', ', $categories) ;
+		}
+		return $categories;
+	}	
     /**
      * @Route("/ajax/email-message/{id}/{match_user_id}", name="admin_ajax_email_message", options={"expose"=true})
      * @Method({"GET"})
      */
     public function ajaxEmailMessageAction(Request $request, User $user)
     {
+	    $locale             = $this->requestStack->getCurrentRequest()->getLocale();
         $matchUser          = $this->userManager->getFind($request->get('match_user_id'));
         $userRequest        = $this->connectionRequestManager->getFindOneOpenByUser($user);
         $matchUserRequest   = $this->connectionRequestManager->getFindOneOpenByUser($matchUser);
-
+		$matchUserCategories = $this->categoryManager->getFindByIdsAndLocale(array_keys($user->getCategoryNames()), 'sv');
+		
+		$userCategories        = $matchUser->getCategoryNames();			
+	 	$categories_sv  =  $this->userManager->getCategoriesExactMatchByUserAndLocale($matchUser , $user, 'sv');
+		$categories_en  =  $this->userManager->getCategoriesExactMatchByUserAndLocale($matchUser , $user, 'en');
+		
         return new JsonResponse([
             'success'               => true,
             'user_message'          => $this->templating->render('email/matchEmail.txt.twig', [
                 'user'      => $user,
                 'matchUser' => $matchUser,
-                'request'   => $matchUserRequest]),
+                'request'   => $matchUserRequest,
+				'categories_EN' => $categories_en,
+				'categories_SV' => $categories_sv]),
             'match_user_message'    => $this->templating->render('email/matchEmail.txt.twig', [
                 'user'      => $matchUser,
                 'matchUser' => $user,
-                'request'   => $userRequest])
+                'request'   => $userRequest,
+				'categories_EN' => $categories_en,
```

Comment:
Please use lowercase. Uppercase in locale means country (and not language).

Reply:
Fixed